qa/mc_sc_connected_nodes.py

```diff
@@ -66,22 +95,25 @@ def run_test(self):
         sc_info_2 = sc_nodes_info[2]
 
         # check mainchain block inclusion for sidechain nodes 1 and 2
-        check_mainchan_block_inclusion(sc_0, sc_info_0[0], 1, 0, mc_0.getblock(str(sc_info_0[3])), sc_info_0[1], True)
```

Comment:
what is the goal of alias declaration for `sc_0`, `mc_0`, `sc_nodes_info`, etc? As for me it decrease readability and understanding of the test flow

Reply:
Renamed